crates/cargo-test-support/src/registry.rs

```diff
@@ -55,12 +60,30 @@ fn generate_url(name: &str) -> Url {
     Url::from_file_path(generate_path(name)).ok().unwrap()
 }
 
+#[derive(Clone)]
+pub enum Token {
+    Bearer(String),
```

Comment:
It's not really a bearer token, since Cargo/crates.io made an implementation error and don't actually prefix the token with `Bearer `.

Reply:
Will be fixed in the next push.